components/x-teaser/src/Image.jsx

```diff
@@ -25,26 +25,30 @@ const LazyImage = ({ src, lazyLoad }) => {
 }
 
 export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, imageHighestQuality, ...props }) => {
-	const displayUrl = relativeUrl || url
-	const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'))
-	const options = imageSize === 'XXL' && imageHighestQuality ? { quality: 'highest' } : {}
-	const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize], options) : image.url
-	const ImageComponent = imageLazyLoad ? LazyImage : NormalImage
+	if (image) {
```

Comment:
`if (!image) { return null }` on the top of this function would be clearer?

Reply:
Oo yes! early return! I love that!! Great suggestion! Thank you 🙌